test/e2e_node/cpu_manager_test.go

```diff
@@ -672,6 +672,56 @@ func runCPUManagerTests(f *framework.Framework) {
 		runAutomaticallyRemoveInactivePodsFromCPUManagerStateFile(ctx, f)
 	})
 
+	ginkgo.It("reservedSystemCPUs are excluded only for Gu pods (strict-cpu-reservation option not enabled by default)", func(ctx context.Context) {
```

Comment:
What about cases corresponding to compatibility of this policy option with other previously proposed policy options? Since the feature is going Beta, I think we need to test if it works as expected with other policy options as well.

We had a discussion about this during review of another KEP (https://github.com/kubernetes/enhancements/pull/5117/files#r1946364747) and it was suggested that we should have a compatibility matrix with other policy options.

Reply:
good point. I think most of this concern can be addressed with unit tests, but likely a few e2e tests with/without this option enabled can be beneficial. We can start with expected compatibility (all policy options I believe)